### PR TITLE
Create log out functionality onClick

### PR DESCRIPTION
**Pull request by nataliemcintyre2021**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- Created log out method to be invoked on button click

## What (if anything) did you refactor?
- Set method in App.js

## Were there any issues that arose?
- e.preventDefault was breaking functionality

## Is there anything that you need from your teammates?
- Review to make sure it doesn't break other methods? Too much prop drilling?

## Any other comments, questions, or concerns?
-

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor

## How Has This Been Tested?
- [ ] open index.html
- [X] console.log()
- [X] dev tools
- [ ] Cypress

## Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by MTCawthray — APPROVED** [review]
Excellent! That is a great UI/UX element to have done. This would be so much easier to accomplish with global state management, but thanks for figuring out the logic to accomplish this without that!

File: src/components/App/App.js [hunk 1]
```diff
@@ -18,12 +19,18 @@ function App() {
   const {error, loading, data} = useQuery(GET_USER, {
     variables: {id: id}
   })
+  const history = useHistory()
```
Comment (hayleyw7) [inline src/components/App/App.js]: This is 100% new to me. Great way to solve this problem.

File: src/components/App/App.js [hunk 2]
```diff
@@ -18,12 +19,18 @@ function App() {
...
 
   useEffect(() => {
 
     setUser(data);
   }, [data])
 
+  const logOut = () => {
```
Comment (hayleyw7) [inline src/components/App/App.js:29]: Great solution.

**Review by hayleyw7 — APPROVED** [review]
Creative and efficient solution here. GREAT work!

**Pull request merged**